build/extract.js

```diff
@@ -0,0 +1,339 @@
+/*!***************************************************
+ * diacritics
+ * http://diacritics.io/
+ * Copyright (c) 2016–2017 The Diacritics Authors
+ * Released under the MIT license https://git.io/vXg2H
+ *****************************************************/
+"use strict";
+const fs = require("fs"), // file system
+  path = require("path"),
+  // delete files using patterns
+  del = require("del"),
+  // remove combining diacritics
+  stripMarks = require("strip-combining-marks"),
+
+  // get metadata.alphabet
+  alphabet = require("cldr-data/supplemental/languageData")
+    .supplemental.languageData,
+  // get metadata.language in English
+  langEn = require("cldr-data/main/en/languages")
+    .main.en.localeDisplayNames.languages,
+
+  // CLDR data path
+  cldrData = "node_modules/cldr-data/",
+  // message added to beginning of generated files
+  unvalidatedMessage = `/**
+ * This file was automatically generated and contains unvalidated content
+ * Once verified, add the base IETF language tag to the array in the
+ * build/validated-languages.json file and remove this comment
+ */
+`;
+
+/**
+ * Extract
+ */
+class Extract {
+  /**
+   * Constructor
+   */
+  constructor() {
+    this._continents = {};
```

Comment:
It was meant to indicate that it's an internal variable and not to be accessed outside of the class.